tests/method/tmultimjs.nim

```diff
@@ -4,7 +4,7 @@ discard """
 Hi derived!
 hello
 '''
-  disabled: true
+    disabled: true
```

Comment:
Disabled iaround 2020-03 in a commit that references #13803, for CI sake?
Working in my pc.